app/entities/account.go

```diff
@@ -0,0 +1,119 @@
+package entities
+
+import (
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Represents a user account in the digital bank.
+type Account struct {
+	id         string
+	name       string
+	cpf        string
+	secret     string
+	balance    int64
```

Comment:
Acho melhor usar `int` até pq vai facilitar na hora de fazer operações matemáticas com esse campo
```suggestion
	balance    int
```